design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,361 @@
+type Query {
+
+    getNegotiableQuote(
+        id: ID!
+    ): NegotiableQuote @doc(description:"Get negotibale quote")
+
+    getAllNegotiableQuotes: [NegotiableQuote]! @doc(description: "Get all negotiable quotes")
+}
+
+type Mutation {
+
+    updatedPrices(
+        updatedQuotes: UpdatedPricesInput!
+    ): UpdatedPricesOutput! @doc(description:" This call refreshes item prices, taxes, discounts, cart rules in the negotiable quote. Quotes that are locked for the seller will not be updated.")
+
+    requestQuote(
+        negotiableQuote: NegotiableQuoteRequestInput!
+    ): RequestQuoteOutput! @doc(description:"Generates negotiable quote request")
+
+    assignBillingAddressToNegotiableQuote(
```

Comment:
Let's align naming with cart functionality: `setBillingAddressOnCart`

Reply:
@mbrinton01 do we even have an ability to set billing address on quote, before proceeding to the checkout, where billing address is set on cart? It seems impossible based on https://docs.magento.com/user-guide/customers/account-dashboard-quotes-negotiate.html

Reply:
No, the billing address cannot be modified in the NQ process.

Reply:
Removed `assignBillingAddressToNegotiableQuote` Mutation.